src/payment.coffee

```diff
@@ -93,6 +85,10 @@ cards = [
   }
 ]
 
+# Store HTML elements (passed onto formatters)
+# for re-use by other methods
+elements = []
```

Comment:
I'm not sure I'm comfortable with this. It's adding state to what should be a stateless library. What are your thoughts on that?